src/Navigation/Notifications/Repository/Notify.php

```diff
@@ -479,7 +482,8 @@ function createFromArray($params)
 						$tsitelink = sprintf($sitelink, $params['link']);
 						$hsitelink = sprintf($sitelink, '<a href="'.$params['link'].'">'.$sitename.'</a><br><br>');
 						break;
-					case "SYSTEM_DB_UPDATE_FAIL":
+
+					case 'SYSTEM_DB_UPDATE_FAIL': // @TODO Unused (only here)
```

Comment:
`@TODO` should be used to indicate code parts that need rework or could be enhanced in the future. I don't a reason for a `@TODO` here. Some `// currently unused` would be more appropriate, I think.

Reply:
The disadvantage is that it (maybe this is a weak example) gets lost and ignored over time.

Reply:
Because of the over usage of `@TODO` I now tend to simply ignore them absolutely.

Reply:
That' not my problem, or is it?

Reply:
Unfortunately, it is your problem. You're writing code not only for the machines that will execute it, but also for the humans working on it alongside you.